package.json

```diff
@@ -7,7 +7,9 @@
     "url": "git://github.com/kubernetes-ui/label-selector.git"
   },
   "main": "labelFilter.js",
-  "dependencies": {},
+  "dependencies": {
+    "grunt-contrib-watch": "^0.6.1"
```

Comment:
actually can you move this line into devDependencies instead, i told you the wrong arg so it put it in the wrong block :)